### PR TITLE
Fix `.gitignore` and extend `ValidatorCleanRestart` timeout

### PR DESCRIPTION
**Pull request by Geod24**
See each commit.

### CODE REVIEW

**Review by AndrejMitrovic — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -15,6 +15,8 @@
 *.exe
 build/
 tracy_build
+source/agora/cli/version/version_build
+source/scpp/cpp_build
```
Comment (kchulj) [inline .gitignore:19]: These files were created when I run `dub build`..

Comment (Geod24) [reply .gitignore:19]: Yes they are binaries. What's interesting is that they were not created before, or we would have seen it. So maybe verbose mode changed something ?

**Review by linked0 — APPROVED** [review]
LGTM

File: .gitignore (hunk 1 above)
Comment (kchulj) [reply .gitignore:19]: Might as well try without the verbose mode.

**Pull request merged**